chosen-docs/upstream/deploying-a-devfile-using-odo-on-IBM-Z-and-Power.md

```diff
@@ -0,0 +1,22 @@
+---
+# Page settings
+layout: default
+keywords:
+comments: false
+
+# Hero section
+title: Title here
+description: Description here
+
+# Micro navigation
+micro_nav: true
+
+# Page navigation
+page_nav:
+    prev:
+        content: Previous page
+        url: '/docs/previous-page'
+    next:
+        content: Next page
```

Comment:
content should be "Devfile file reference"
and url "/file-reference/"

Reply:
Also change the "next" section in `chosen-docs/upstream/deploying-a-devfile-using-odo.md` file so it points to this new document and not the file-reference anymore